manifests/charts/istio-cni/values.yaml

```diff
@@ -19,6 +19,7 @@ cni:
   # defaults:
   cniBinDir: "" # Auto-detected based on version; defaults to /opt/cni/bin.
   cniConfDir: /etc/cni/net.d
+  cniAmbientConfigDir: "" # defaults to /etc/ambient-config
```

Comment:
Should we keep this in the ambient setting field?

Reply:
thanks, I had moved it in ambient setting